test_integration/test_ee_scaling_spin_mix.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
```

Comment:
This is just a copy of the steam dgemm mix py file... Needs to be updated with the asserts in my integration test

Reply:
Removed top commit from this PR